x-pack/plugins/ml/server/routes/schemas/data_analytics_schema.ts

```diff
@@ -60,6 +60,13 @@ export const analyticsIdSchema = schema.object({
   analyticsId: schema.string(),
 });
 
+export const analyticsIdIndexSchema = schema.object({
+  /**
+   * Analytics ID
+   */
```

Comment:
Probably a copy-paste issue. Should be `Destination index` I guess

Reply:
Fixed in https://github.com/elastic/kibana/pull/66934/commits/d14fafab86d64ef3afce4bda0f3c415ea85e672e